devops/templates/test-run-step-template.yml

```diff
@@ -106,6 +106,7 @@ steps:
 
 - bash: |
       source activate ${{parameters.condaEnv}}
-      python -m pip install flake8
+      python -m pip install flake8, isort
       flake8 --max-line-length=119 --exclude=.git/,__pycache__/,dist/ .
-  displayName: 'Run flake8'
+      isort . -c
```

Comment:
I think you should set --line-length=119 to be consistent with above setting?

Reply:
well, maybe this is fine as is.  It's probably better to just keep the defaults.